ui/Screen/Bsod.svelte

```diff
@@ -14,29 +21,71 @@
   }
 
   .container {
+    height: 100%;
+    padding: 5rem;
+
     display: flex;
-    text-align: center;
     justify-content: center;
     align-items: center;
     flex-direction: column;
     color: #fff; /* I know... but this design doesn't work in dark mode. */
   }
+  .proxy-log-container {
+    background: #e3e3ff;
+
+    max-width: 100%;
+    min-width: 40em;
+    max-height: 20vh;
+    overflow: scroll;
+
+    margin-top: 2rem;
+    border-radius: 8px;
+    padding: 0.9em;
+  }
+
+  .proxy-log {
+    display: block;
+    color: #5555ff;
+    font-size: 14px;
+    white-space: pre-wrap;
+    line-height: 1.4;
+
+    max-width: 100%;
+  }
 </style>
 
-{#if $fatalError}
+{#if $fatalError !== null}
   <Fullscreen
     escapable={false}
-    style="background-color: var(--color-secondary); z-index: 200;"
-    contentStyle="display: flex; width: 312px; align-items: center;">
+    style="background-color: var(--color-secondary); z-index: 200;">
     <div class="container" data-cy="blue-screen-of-death">
       <Emoji emoji="🧻" size="huge" style="margin-bottom: 1.5rem;" />
-      <p>
-        We're not totally sure what's going on, but we can't load the app right
-        now because the background process won't start up.
+      <p style="width: 321px; text-align: center">
+        {#if $fatalError.kind === 'SESSION'}
+          We're not totally sure what's going on, but we can't load the app
+        {:else if $fatalError.kind === 'PROXY_EXIT'}
+          Hmm, looks like the app can’t be loaded right now because the backend
+          has crashed or it isn’t starting.
+        {/if}
       </p>
       <Button style="display: flex; color: #fff;" on:click={tweet}>
         Reach out for support
       </Button>
+      {#if $fatalError.kind === 'PROXY_EXIT' && $fatalError.data.stderr}
+        <div class="proxy-log-container">
```

Comment:
I think this was supposed to be a `<Textarea>` so that you could scroll around the box to see more of the error logs. If we keep the `<div>`, maybe adding `overflow: scroll;` would be enough?

Reply:
`overflow: scroll` [is set](https://github.com/radicle-dev/radicle-upstream/pull/1216/files#diff-12eb7ad23892822d0d696e8897826d85a5edf99cb0ccc1a3fe8898a813e1da20R39) for `.proxy-log-container` and it is indeed scrollable. Is there an advantage to using `Textarea`?

Reply:
Ah, cool. Couldn't check when I tested before. Will test in next pass.

Reply:
Yeah, no need for `Textarea`, scrolling works just find.

Could we add a notification when the logs are copied to clipboard, like we have it in other places in the app?

<img width="297" alt="Screenshot 2020-11-11 at 12 41 02" src="https://user-images.githubusercontent.com/158411/98807624-39f4e600-241b-11eb-948d-56b972988d1e.png">

Reply:
Fixed in https://github.com/radicle-dev/radicle-upstream/pull/1216/commits/ba90d3f385a2f10faa83d2240db4b4a194c8f460